src/pages/ContentScripts/features/repo-header-labels/view.tsx

```diff
@@ -37,6 +37,27 @@ const View = ({ activity, openrank, participant, contributor, meta }: Props): JS
   const participantData = generateDataByMonth(participant, meta.updatedAt);
   const contributorData = generateDataByMonth(contributor, meta.updatedAt);
 
+  type Label = {
+    id: string;
+    name: string;
+    type: string;
+  };
+
+  const labels: Label[] = meta.labels as Label[];
```

Comment:
Not all repositories have label data, so it is best to add statements that determine the existence of `meta.labels`. Otherwise, it will result in an error and the previous` hypercrx-repo-header-labels` will not be displayed. For example, in this repository:[Travel_Website](https://github.com/apu52/Travel_Website).
<img width="1280" alt="image" src="https://github.com/user-attachments/assets/c62db1a6-c90a-499c-b0d8-0e2c70678e70">